ee/vellum_ee/workflows/display/nodes/vellum/base_adornment_node.py

```diff
@@ -37,3 +41,46 @@ def serialize(
         serialized_wrapped_node["adornments"] = adornments + [adornment] if adornment else adornments
 
         return serialized_wrapped_node
+
+    @classmethod
+    def wrap(cls, **kwargs: Any) -> Callable[..., Type[BaseNodeDisplay]]:
```

Comment:
We lose a mypy benefit here, which is fine since:
- Display classes are codegen only for now
- We are deferring all mypy improvements to at least Q3